app/views/categories/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<ol class="breadcrumb">
+  <li>
+    <a href="/categories">Categories</a>
```

Comment:
you should leverage your rails helpers here and use link_to - also, it's good practice to always use your path helpers rather than hardcoding "/companies"